lxd/instance/drivers/qmp/monitor.go

```diff
@@ -19,17 +19,20 @@ var monitorsLock sync.Mutex
 // RingbufSize is the size of the agent serial ringbuffer in bytes.
 var RingbufSize = 16
 
+// AgentStatusStarted is the event sent once the lxd-agent has connected successfully.
+var AgentStatusStarted = "LXD-AGENT-STARTED"
```

Comment:
Also I think we should avoid using word "connected" here as that is a separate status.